multimodal/src/autogluon/multimodal/optimization/utils.py

```diff
@@ -799,3 +799,16 @@ def compute_probability(
         prob = 1 - prob
 
     return prob
+
+# Reference: https://github.com/lenscloth/RKD/blob/0a6c3c0c190722d428322bf71703c0ae86c25242/metric/utils.py#L6
+def pdist(e, squared=False, eps=1e-12):
```

Comment:
Docstrings + type hints

Reply:
`e` is not a good variable name. Meaningful variable/function names are necessary for good code quality.

Reply:
Changed it to embeddings